commandServer/routes.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+
+class BaseModel(object):
+
+    def get(self, request, params=None):
+        return json.dumps([]).encode('utf_8')
+
+    def post(self, request, params=None):
+        return json.dumps([]).encode('utf_8')
+
+    def put(self, request, params=None):
+        return json.dumps([]).encode('utf_8')
+
+    def delete(self, request, params=None):
+        return json.dumps([]).encode('utf_8')
+
+
+ROUTES = {r'/': BaseModel,
```

Comment:
For me it's not suitable here. We have @asyncio.coroutine, and adding one more it's not the best solution.